api/v1alpha1/nonadminbackup_types.go

```diff
@@ -43,6 +45,18 @@ type NonAdminBackupSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=trace;debug;info;warning;error;fatal;panic
 	LogLevel string `json:"logLevel,omitempty"`
+
+	// DeleteBackup tells the controller to remove created Velero Backup
```

Comment:
I think is important to doc that this deletes storage data as well

Reply:
`DeleteBackup tells the controller to create DeleteBackupRequest which removes all backup data including backup storage and Velero Backup objects`

Is the above fine?

Reply:
I would not tell about implementation detail (DeleteBackupRequest) to user, but it is ok, the message contains the important part

Reply:
Ok will reword this to be aligned with https://github.com/vmware-tanzu/velero/blob/main/site/content/docs/main/backup-reference.md#deleting-backups

Reply:
Done please check newest commit.